addons/disable-stage-drag-select/addon.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Disable sprite dragging in editor",
+  "description": "Makes dragging sprites in the project player behave exactly as in fullscreen while in the editor.",
```

Comment:
This description is a bit weird:
- it sounds like you can still drag sprites in full screen, but differently;
- "stage" would be a more appropriate word than "project player" here because it's talking about sprites.

Maybe "Removes the ability to drag sprites around in the editor"?

Reply:
What about the name? I thought we would stop using verbs?